slow_tests/test_determinism.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+
+from ff.handlers import openmm_deserializer
+from ff.handlers.deserialize import deserialize_handlers
+from ff import Forcefield
+
+from simtk.openmm import app
+
+from timemachine.lib import custom_ops, LangevinIntegrator
+
+from fe.utils import to_md_units
+from fe.model_utils import apply_hmr
+from fe import free_energy
+from fe.topology import SingleTopology
+
+from md import builders, minimizer
+from md.barostat.utils import get_bond_list, get_group_indices
+from testsystems.relative import hif2a_ligand_pair as testsystem
+
+def test_deterministic_energies():
```

Comment:
Add one-liner docstring describing assertion. (Test asserts something stronger than just deterministic energies -- "energies recomputed using bound impls on the snapshots returned by `multiple_steps` agree bitwise with the energies returned by `multiple_steps`" or similar.)

Reply:
We may be interested in generalizing the test to assert also consistency between `execute`, `execute_selective`, and `execute_fixed` (although some of these comparisons will require picking a tolerance, rather than being bitwise).

Reply:
Agreed, would be good to test all three.

Reply:
Dug in a bit and realized there is more of a difference than I had thought.